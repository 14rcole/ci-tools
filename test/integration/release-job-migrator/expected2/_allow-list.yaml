periodic-ci-openshift-release-master-ci-4.6-e2e-aws: blocking
release-openshift-origin-installer-e2e-gcp-4.7: blocking
release-openshift-origin-installer-e2e-gcp-upgrade-4.7: informing
release-openshift-origin-installer-e2e-aws-compact-4.7: informing
