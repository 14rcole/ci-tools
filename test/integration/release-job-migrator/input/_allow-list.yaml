release-openshift-origin-installer-e2e-aws-4.6: blocking
release-openshift-origin-installer-e2e-gcp-4.7: blocking
release-openshift-origin-installer-e2e-gcp-upgrade-4.7: informing
release-openshift-origin-installer-e2e-aws-compact-4.7: informing
