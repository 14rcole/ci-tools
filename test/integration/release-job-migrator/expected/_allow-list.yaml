periodic-ci-openshift-release-master-origin-4.6-e2e-aws: blocking
release-openshift-origin-installer-e2e-gcp-4.7: blocking
periodic-ci-openshift-release-master-origin-4.7-e2e-gcp-upgrade: informing
release-openshift-origin-installer-e2e-aws-compact-4.7: informing
